app/ssttpeligibility/EligibilityService.scala

```diff
@@ -140,4 +142,14 @@ object EligibilityService {
         true
     }
   }
+
+  private def isDatePresentAndAfterToday(today: LocalDate, dateOption: Option[LocalDate]): Boolean = {
+    dateOption match {
```

Comment:
Could you simplify the code to one line please: `dateOption.exist(_.isAfter(today))`